src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -42,6 +44,129 @@
 public final class RqMultipartTest {
 
     /**
+     * Carriage return constant.
+     */
+    private static final String CR = "\r\n";
+    /**
+     * Two RqMultipart objects are equal when inited with equal Request objects.
+     * @throws IOException if some problem inside
+     */
+    @Test
+    public void equalsReturnsTrue() throws IOException {
```

Comment:
Thanks for the link.

But after reading I can't see what would be correct name using "RqMultipart can .." pattern:
"RqMultipart can equals to the same data inited RqMultipart" as for the doc and
equalsToTheSameDataInitedRqMultipart as for method name.

Not sure it sounds good enough.
Could you please provide your ideas?